api/objectstore/minio.go

```diff
@@ -0,0 +1,99 @@
+package objectstore
+
+import (
+  "github.com/minio/minio-go/v7"
+  "github.com/minio/minio-go/v7/pkg/credentials"
+  "github.com/charmbracelet/log"
+  "io"
+  "os"
+  "strconv"
+  "context"
+)
+
+const (
+  SALZ_DEFAULT_BUCKET = "salz"
```

Comment:
env var?

Reply:
we already look for the value for the bucket in the `SALZ_BUCKET` environment variable, this is if that is not found.